server/service/src/preference/mod.rs

```diff
@@ -26,11 +26,14 @@ pub trait PreferenceServiceTrait: Sync + Send {
         let PreferenceProvider {
             show_contact_tracing,
             display_population_based_forecasting,
+            allow_tracking_of_received_stock_by_donor,
         } = self.get_preference_provider();
 
         let all_prefs_descriptions = vec![
             // Add each pref here
             show_contact_tracing.as_description(connection, store_id.clone())?,
+            allow_tracking_of_received_stock_by_donor
+                .as_description(connection, store_id.clone())?,
```

Comment:
Apologies, this section changed again, with merge of store pref support 🫠  Hopefully ok to resolve in merge, hit me up if conflicts are funky 😁